### DISCUSSION TITLE
How to compare Json with different case in property name?

### DISCUSSION THREAD

**Discussion by mflay1** [fluentassertions/fluentassertions.json#68] [General]
Hi, 

Is it possible to compare properties ignoring case sensitivity?
Example:
``` cs
var actual = JToken.Parse(@"{ ""KEY1"" : ""value"" }"); // uppercase property
var expected = JToken.Parse(@"{ ""key1"" : ""value"" }"); // lowercase property
actual.Should().BeEquivalentTo(expected);
```
I couldn't find any info about this. Also chat is inaccessible for me.

**Comment by dennisdoomen** [comment]
Unfortunately that is not possible. It would be a nice feature though, but we don't have anything planned for that.
